### PR TITLE
Add Kdl.NetStandard.x64 (Fix Robots new missing dependency error)

### PR DESCRIPTION
**Pull request by bdovaz**
Fixes a problem generated by this new dependency of `Robots`.

image

cc: @xoofx

### CODE REVIEW

**Comment by bdovaz** [comment]
Friendly ping @xoofx :)

**Comment by xoofx** [comment]
@bdovaz sorry, I'm still on holidays, didn't have access to a computer until yesterday. I have added you as a collaborator to this repository so that you should be able to merge PR. I will have to double check if you can create tags as well, but maybe, and in that case, I would be less blocking stuffs. 😅

**Comment by bdovaz** [comment]
@xoofx a doubt, on the subject of tags.

From what I see in the workflow for it to be uploaded to Azure I have to create a tag. I see that I would have to follow these steps, no?

https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository#creating-a-release

I mean, is it done by going here? https://github.com/xoofx/UnityNuGet/releases/new

Or is it done in another way? Because I see that you don't have any release created, just tags.

In short, I would like you to explain me the flow to get a new release uploaded to Azure.

**Comment by xoofx** [comment]
Yeah, it's just a regular git tag, so you can create them either from the web UI or from the cmd line, I'm not using git releases. But it could be that you won't be able to push a tag, in that case I will do it.

**Comment by bdovaz** [comment]
@xoofx and how to create a tag through web UI? Should I see a button on this page? I only see the list of tags.

https://github.com/xoofx/UnityNuGet/tags

I'm curious more than anything else, although I usually do these operations from my git client.

**Comment by xoofx** [comment]
Only releases can be created from the web UI but it should create a tag
